offline/offline_tools/matlab_hax/sensor_data_plotting/code_src/main.m

```diff
@@ -1,193 +0,0 @@
-% code to take sensor data and map it out on a graph
```

Comment:
I am not 100% sure, I just could not figure out where to put them so I cut them.  We should keep them in history incase that was a bad move